node/nodeRunner.go

```diff
@@ -904,7 +901,6 @@ func (nr *nodeRunner) CreateManagedProcessComponents(
 		ImportStartHandler:     importStartHandler,
 		WorkingDir:             configs.FlagsConfig.WorkingDir,
 		HistoryRepo:            historyRepository,
-		OutportHandler:         managedStatusComponents.OutportHandler(),
```

Comment:
a lot of empty files in outport/drivers/elastic directory

Reply:
these files were removed.

Reply:
ok